tests/TestCase/Core/AppTest.php

```diff
@@ -216,6 +216,22 @@ public function shortNameProvider()
         ];
     }
 
+    /**
+     * test classPath() with a plugin.
+     *
+     * @return void
+     */
+    public function testClassPathWithPlugins()
+    {
+        $basepath = TEST_APP . 'Plugin' . DS;
+        $this->loadPlugins(['TestPlugin', 'Company/TestPluginThree']);
+        $result = App::classPath('Controller', 'TestPlugin');
+        $this->assertPathEquals($basepath . 'TestPlugin' . DS . 'src' . DS . 'Controller' . DS, $result[0]);
```

Comment:
Nitpick: A newline after this assert would have been nice for easier reading :slightly_smiling_face:.

Reply:
True. I just copied them from the other test case.